bin/get-secrets

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
+'use strict';
+const fs = require('fs');
+const assert = require('assert');
+const AWS = require('aws-sdk');
+
+AWS.config.update({ region: 'eu-west-2' });
+
+const secretDir = '/etc/craft';
+const parametersDest = secretDir + '/parameters.json';
+const envFileDest = '/var/www/craft/.env';
+
+const argv = require('yargs').option('environment', {
+    alias: 'env',
+    default: 'development',
+}).argv;
+
+const secretsManager = new AWS.SecretsManager();
+
+const getSecret = secretId => {
+    return new Promise((resolve, reject) => {
```

Comment:
FWIW I think you can use `secretsManager.getSecretValue.promise()` for an inbuilt promise version of this method.